File: android-exercises/2 - dialog/AlertDialog_java/app/src/main/java/com/bugrahankaramollaoglu/alertdialog_java/MainActivity.java
package com.bugrahankaramollaoglu.alertdialog_java;

import android.app.AlertDialog;
import android.content.DialogInterface;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    Button button;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        button = (Button) findViewById(R.id.button);

    }

    public void click(View view) {
        alertDialog();
    }

    public void alertDialog() {
        AlertDialog.Builder builder = new AlertDialog.Builder(this);
        builder.setTitle("alert dialog");
        builder.setMessage("this is deneme");

        // positive
        builder.setPositiveButton("Yes", new DialogInterface.OnClickListener() {
            @Override
            public void onClick(DialogInterface dialog, int which) {
                Toast.makeText(getApplicationContext(), "you said yes", Toast.LENGTH_SHORT).show();
            }
        });

        // negative
        builder.setNegativeButton("No", new DialogInterface.OnClickListener() {
            @Override
            public void onClick(DialogInterface dialog, int which) {
                Toast.makeText(getApplicationContext(), "you said no", Toast.LENGTH_SHORT).show();
            }
        });

        AlertDialog dialog = builder.create();
        dialog.show();

    }
}

File: android-exercises/2 - dialog/AlertDialog_kotlin/app/src/main/java/com/bugrahankaramollaoglu/alertdialog_kotlin/MainActivity.kt
package com.bugrahankaramollaoglu.alertdialog_kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.Toast
import androidx.appcompat.app.AlertDialog

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val button = findViewById<Button>(R.id.button)
    }

    fun clicked(view: View) {
        alertDialog()
    }

    private fun alertDialog() {
        val builder = AlertDialog.Builder(this)
        builder.setTitle("alert dialog")
        builder.setMessage("this is deneme")

        // positive
        builder.setPositiveButton("Yes") { dialog, which ->
            Toast.makeText(this, "you said yes", Toast.LENGTH_SHORT).show()
        }

        // negative
        builder.setNegativeButton("No") { dialog, which ->
            Toast.makeText(this, "you said no", Toast.LENGTH_SHORT).show()
        }

        val dialog: AlertDialog = builder.create()
        dialog.show()
    }
}

